src/Components/Server/src/ComponentHub.cs

```diff
@@ -225,6 +236,16 @@ private async void CircuitHost_UnhandledException(object sender, UnhandledExcept
             }
         }
 
+        private bool IsBadInput(Exception exception)
+        {
+            return exception switch
+            {
+                InvalidEventException _ => true,
+                RemoteRendererException { BadInput: true } => true,
+                _ => false
+            };
```

Comment:
```C#
while (true)
{
    pranavPoints++;
}
```

Reply:
TBH I find this pattern of checking for specific types to be an issue in the long run. When people try to implement a new feature following the same pattern (e.g., defining a new exception type for invalid input, equivalent to `InvalidEventException`), there's nothing to force them to discover the need to also add it to the list here. By default they won't, and then that type will get logged as a warning. Even if they know they have to do something to whitelist their new exception type, there's no way of pressing F12 to follow the trail here.

If we had a common base type for "invalid input exception" this problem would disappear, as anyone copy-pasting the same approach as an existing bit of code would naturally copy the same base type for their new exception type.

Reply:
Another idea here would be to use a different kind of event args in our unhandled exception event. Then we could pass in whethere the execption was based on input, or background. It doesn't require defining new exception types, just a new eventargs type.

Reply:
For context, I discovered that .NET already has an `UnhandledExceptionEventArgs` which is what we're using. This could just be it's own type with all of the info we need.

Reply:
I believe we cover all the cases where we process input already. This wouldn't be a feature for customers to use if that's what it's implied here.

Reply:
This is also obsolete and has its own custom exception type.